src/main/java/net/geforcemods/securitycraft/items/CodebreakerItem.java

```diff
@@ -14,6 +26,28 @@ public CodebreakerItem() {
 		super(new Item.Properties().maxStackSize(1).defaultMaxDamage(3).group(SecurityCraft.groupSCTechnical));
 	}
 
+	@Override
+	public ActionResult<ItemStack> onItemRightClick(World world, PlayerEntity player, Hand hand) {
+		ItemStack codebreaker = player.getHeldItem(hand);
+
+		if (!world.isRemote) {
+			if (hand == Hand.MAIN_HAND && player.getHeldItemOffhand().getItem() == SCContent.BRIEFCASE.get()) {
+				ItemStack briefcase = player.getHeldItemOffhand();
```

Comment:
You are not using this variable unless the codebreaker works, meaning this wastes resources (if not much) when the codebreaker is not enabled or it fails to break the code. Move it down into the else if blockm, please. Also no need to create a variable if you're only using it once in the following code.